x-pack/plugins/transform/server/routes/api/transforms.ts

```diff
@@ -439,6 +445,27 @@ export function registerTransformsRoutes(routeDependencies: RouteDependencies) {
     license.guardApiRoute<undefined, undefined, StopTransformsRequestSchema>(stopTransformsHandler)
   );
 
+  /**
+   * @apiGroup Transforms
+   *
+   * @api {post} /api/transform/schedule_now_transforms Schedules transforms now
```

Comment:
Please can you add an API integration test for this new route.

Reply:
Added API integration tests in 3563f145bf2.